### ISSUE TITLE
New API functions?

### ISSUE THREAD

**Issue by ryobg** [ryobg/JContainers#36]
I will drop here some notes for functions which may be of use for JC users.

- [ ] `int JArray.erase[Integers|Floats|Strings|Forms|Objects] (object, value, start = 0, end = -1)`  - basically run over the whole range and delete all items which equal the passed value. Optionally, give specify sub-range. Return the number of erased items.

**Comment by RealAntithesis** [comment]
I assume you are looking for c/c++ functions to go into the dll for direct access in papyrus? I've done similar functions in lua (being much faster than papyrus) but the lua interface is a bit indirect (but still more than useful once I got it into my workflow).

**Comment by ryobg** [comment]
@RealAntithesis yes, it is about the DLL API, which can be easily translated to Papyrus and Lua.
